src/cow-react/common/hooks/useCancelOrder/index.ts

```diff
@@ -47,12 +46,12 @@ export function useCancelOrder(): (order: Order) => UseCancelOrderReturn {
     (order: Order) => {
       // Check the 'cancellability'
 
-      const isEthFlowOrder = getIsEthFlowOrder(order)
+      // const isEthFlowOrder = getIsEthFlowOrder(order)
 
       // 1. EthFlow orders will never be able to be cancelled offChain
       // 2. The wallet must support offChain singing
       // 3. The order must be PENDING
-      const isOffChainCancellable = !isEthFlowOrder && allowsOffchainSigning && order?.status === OrderStatus.PENDING
+      const isOffChainCancellable = true
```

Comment:
@shoom3301 ❓❓❓
i don't get this change

Reply:
I've tested with it, my bad.
https://github.com/cowprotocol/cowswap/issues/2392 - there is a fix

Reply:
Ahh, ok it was a test